### PR TITLE
OLS-1106: Updated README.md accordingly

### PR DESCRIPTION
**Pull request by tisnik**
## Description

[OLS-1106](https://issues.redhat.com//browse/OLS-1106): Updated README.md accordingly

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [ ] CI configuration change
- [ ] Konflux configuration change

## Related Tickets & Documents

- Related Issue #[OLS-1106](https://issues.redhat.com//browse/OLS-1106)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -459,7 +459,26 @@ Depends on configuration, but usually it is not needed to generate or use API ke
 - `ciphers` is list of enabled ciphers. The values are not checked.
 
 Please look into `examples` folder that contains `olsconfig.yaml` with filled-in TLS security profile for the service.
-Additionally the TLS security profile can be set for any configured provider. In this case the `tlsSecurityProfile` needs to be added into the `olsconfig.yaml` file into `llm_provieders/{selected_provider}` section.
+Additionally the TLS security profile can be set for any configured provider. In this case the `tlsSecurityProfile` needs to be added into the `olsconfig.yaml` file into `llm_provieders/{selected_provider}` section. For example:
```
Comment (JoaoFula) [inline README.md:462]: I know you didn't change this part but llm_providers*

File: README.md [hunk 2]
```diff
@@ -459,7 +459,26 @@ Depends on configuration, but usually it is not needed to generate or use API ke
...
+      - name: gpt-4o-mini
+    tlsSecurityProfile:
+      type: Custom
+      ciphers:
+          - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+          - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+      minTLSVersion: VersionTLS13
+```
+
+Note: the `tlsSecurityProfile` is fully optional. Whet it is not specified, the LLM call won't be affected by specific SSL/TLS settings.
```
Comment (JoaoFula) [inline README.md:481]: **When** it is not specified...

File: README.md (hunk 1 above)
Comment (tisnik) [reply README.md]: fixed, thank you

File: README.md (hunk 2 above)
Comment (tisnik) [reply README.md]: fixed, thank you

**Comment by tisnik** [comment]
/override "Red Hat Konflux / lightspeed-service-on-pull-request"

**Pull request merged**